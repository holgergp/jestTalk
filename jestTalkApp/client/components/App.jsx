import React from 'react';
import List from './ListComponent';
const App = class App extends React.Component {
  render() {
    const myArray = ['1', '2', '3', '4'];
    return (
      <div style={{ textAlign: 'center' }}>
        <h1>Hello World</h1>
        <List values = {myArray} />
      </div>
    );
  }
};
export default App;
