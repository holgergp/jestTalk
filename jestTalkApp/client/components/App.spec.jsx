import App from './App.jsx';
import React from 'react';
import renderer from 'react-test-renderer';

describe('App', () => {
  it('renders', () => {
    const renderedApp = renderer.create(<App />).toJSON();
    expect(renderedApp).toMatchSnapshot();
  });
});
