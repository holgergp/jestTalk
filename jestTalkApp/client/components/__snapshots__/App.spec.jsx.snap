// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<div
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <h1>
    Hello World
  </h1>
  <ul>
    <li>
      <div
        className="Text"
      >
        1
      </div>
    </li>
    <li>
      <div
        className="Text"
      >
        2
      </div>
    </li>
    <li>
      <div
        className="Text"
      >
        3
      </div>
    </li>
    <li>
      <div
        className="Text"
      >
        4
      </div>
    </li>
  </ul>
</div>
`;
